src/Plugin/Filter/EntityEmbedFilter.php

```diff
@@ -102,13 +104,24 @@ public function process($text, $langcode) {
             }
 
             $access = $entity->access('view', NULL, TRUE);
-            $access_metadata = CacheableMetadata::createFromObject($access);
-            $entity_metadata = CacheableMetadata::createFromObject($entity);
```

Comment:
How is it ok to remove the metadata about the entity itself? Are we assuming that every field formatter that can be re-used to display an embedded entity correctly has this metadata in its render API array?